set_env_variables.sh

```diff
@@ -34,10 +43,10 @@ sleep 1
 ARCH=$( uname -m )
 if [ "${ARCH}" = "aarch64" ]; then
    python3 $SCRIPT_DIR/utils/setup/gen_ld_preload.py
-   LD_PRELOAD=$(<$SCRIPT_DIR/utils/setup/.ld_preload)
-   export LD_PRELOAD=$LD_PRELOAD
+   LD_PRELOAD=`cat $SCRIPT_DIR/utils/setup/.ld_preload`
    echo "LD_PRELOAD=$LD_PRELOAD"
 fi
+export LD_PRELOAD=$LD_PRELOAD
```

Comment:
This will fail on x86 because LD_PRELOAD will be undefined variable now.

Reply:
actually, why do we need this condition? Wouldn't it be the same for all platforms?

Reply:
No, we dont need LD_PRELOAD hack on x86

Reply:
so this will not fail, it will do nothing, so it could remain as is